### PR TITLE
docs: snowflake cdc to kafka example

### PR DESCRIPTION
**Pull request by snandam**
This example demonstrates sending Snowflake's Change Data Capture to a private kafka server

### CODE REVIEW

File: examples/command/portals/snowflake/example-5/README.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+  MAX_NODES = 5
+  INSTANCE_FAMILY = CPU_X64_XS;
+
+-- WAIT
+DESCRIBE COMPUTE POOL CDC_TEST_CP;
+
+-- CREATE IMAGE REPOSITORY
+CREATE IMAGE REPOSITORY IF NOT EXISTS CDC_TEST_REPO;
+
+--Note repository_url value to be used to build and publish consumer image to snowflake
```
Comment (etorreborre) [inline examples/command/portals/snowflake/example-5/README.md]: s/--Note/-- Note

File: examples/command/portals/snowflake/example-5/README.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+-- Insert records
+INSERT INTO CDC_TEST_TABLE
+SELECT UUID_STRING(), randstr(255, RANDOM())
+FROM TABLE(GENERATOR(ROWCOUNT => 100));
+
+-- Verify Test Table and CDC Table
+SELECT * FROM CDC_TEST_TABLE;
+SELECT * FROM CDC_TEST_TABLE_STREAM;
+
+-- Publisher checks for changes every 30 seconds. Looks at logs
```
Comment (etorreborre) [inline examples/command/portals/snowflake/example-5/README.md]: In the comments we could add an example of what an expected log line looks like.

File: examples/command/portals/snowflake/example-5/README.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+CALL SYSTEM$GET_SERVICE_LOGS('SNOWFLAKE_CDC_KAFKA_BRIDGE', '0', 'publisher', 100);
+
+-- Check for messages delivered to Kafka
```
Comment (etorreborre) [inline examples/command/portals/snowflake/example-5/README.md]: We could add in the comment that the user can look at the `Verify messages` section below explains how to check the messages.

File: examples/command/portals/snowflake/example-5/README.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+cd -
+```
+- Local machine
+
+```sh
+rm inlet.ticket outlet.ticket
+```
+
+```sh
+# Run if local cluster was created
```
Comment (etorreborre) [inline examples/command/portals/snowflake/example-5/README.md]: s/if local cluster/if a local cluster

File: examples/command/portals/snowflake/example-5/README.md (hunk 2 above)
Comment (etorreborre) [inline examples/command/portals/snowflake/example-5/README.md]: s/Looks at logs/Look at the logs and check that changes have been published?

File: examples/command/portals/snowflake/example-5/README.md [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+> [!IMPORTANT]
+> Replace `TODO` values in `VALUE_LIST` with the output of `ockam project show --jq .egress_allow_list` command in previous step.
+
+```sh
+#Example
+VALUE_LIST = ("k8s-XXX.amazonaws.com:4XXX","k8s-XXX.amazonaws.com:4XXX");
+```
+
+> [!IMPORTANT]
+> Replace `<OCKAM_ENROLLMENT_TICKET>` with contents of `inlet.ticket` generated in previous step
```
Comment (etorreborre) [inline examples/command/portals/snowflake/example-5/README.md]: s/with contents/with the contents

File: examples/command/portals/snowflake/example-5/README.md [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+ockam project ticket --usage-count 1 --expires-in 10h --attribute snowflake-kafka-outlet --relay kafka > outlet.ticket
+# Print the egress allow list for your Ockam project. You will use them later in the tutorial
+ockam project show --jq .egress_allow_list
+
+```
+## Choose between creating an Amazon MSK vs Kafka cluster running on local machine
+
+### Setup Amazon MSK, or
+
+- Run Cloudformation template to create private Amazon Managed Kafka cluster and an EC2 machine running ockam
```
Comment (etorreborre) [inline examples/command/portals/snowflake/example-5/README.md]: s/to create private/to create a private

File: examples/command/portals/snowflake/example-5/README.md [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+[Signup for Ockam](https://www.ockam.io/signup) and then run the following commands on your workstation:
+
+```sh
+# Install Ockam Command
+curl --proto '=https' --tlsv1.2 -sSfL https://install.command.ockam.io | bash && source "$HOME/.ockam/env"
+
+# Enroll with Ockam Orchestrator.
+ockam enroll
+
+# Create enrollment ticket for the node that will run inside container services.
```
Comment (etorreborre) [inline examples/command/portals/snowflake/example-5/README.md]: s/Create enrollment ticket/Create an enrollment ticket

**Review by etorreborre — DISMISSED** [review]
I've added a few comments to add articles because I think it would read better.

**Review by etorreborre — DISMISSED** [review]

**Comment by mrinalwadhwa** [comment]
@snandam the PR has some unrelated changes to a file in the typescript implementation. Let's remove that before we merge

**Review by etorreborre — APPROVED** [review]

**Review by etorreborre — APPROVED** [review]

**Pull request merged**